osu.Game/Online/API/Requests/GetUserBeatmapsRequest.cs

```diff
@@ -11,24 +11,27 @@ public class GetUserBeatmapsRequest : APIRequest<List<APIBeatmapSet>>
     {
         private readonly long userId;
         private readonly int offset;
+        private readonly int limit;
         private readonly BeatmapSetType type;
 
-        public GetUserBeatmapsRequest(long userId, BeatmapSetType type, int offset = 0)
+        public GetUserBeatmapsRequest(long userId, BeatmapSetType type, int offset = 0, int limit = 0)
```

Comment:
why 0? it should be 5 or at least 1 by default (same applied to all requests below)